data_handler/migrations/versions/2dfa022f3110_add_eventbasemodel_and_zklend_event_.py

```diff
@@ -0,0 +1,160 @@
+"""Add EventBaseModel and ZkLend event models
+
+Revision ID: 2dfa022f3110
+Revises: 64a870953fa5
+Create Date: 2024-10-28 18:13:14.699281
+
+"""
+
+import logging
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects.postgresql import ENUM, UUID
+from sqlalchemy.engine.reflection import Inspector
+
+from shared.constants import ProtocolIDs
+
+revision = "2dfa022f3110"
+down_revision = "64a870953fa5"
+branch_labels = None
+depends_on = None
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def upgrade() -> None:
+    """
+    Perform the upgrade migration to create the 'event_base_model', 'accumulators_sync_event_data',
+    and 'liquidation_event_data' tables if they do not exist.
+
+    This migration creates the following tables:
+    - `event_base_model`: Stores general event data with columns for event name, block number, and protocol ID.
+    - `accumulators_sync_event_data`: Inherits from the base model and adds specific fields for token and accumulator values.
+    - `liquidation_event_data`: Inherits from the base model and adds fields specific to liquidation events.
+
+    Additional configuration:
+    - Primary key constraint on the `id` column.
+    - Indexes on fields like `event_name`, `block_number`, and `protocol_id` for optimized querying.
+
+    This function is part of the Alembic migration and is auto-generated.
+    Adjustments may be made if additional configuration or constraints are needed.
+    """
+
+    bind = op.get_bind()
+    inspector = Inspector.from_engine(bind)
+
+    if "event_base_model" not in inspector.get_table_names():
```

Comment:
This model is abstract, so it shouldn't create new table

Reply:
Read it here: 
https://docs.sqlalchemy.org/en/13/orm/extensions/declarative/mixins.html#augmenting-the-base